sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -105,7 +113,15 @@ case class DateAdd(startDate: Expression, days: Expression)
  */
 @ExpressionDescription(
   usage = "_FUNC_(start_date, num_days) - Returns the date that is num_days before start_date.",
-  extended = "> SELECT _FUNC_('2016-07-30', 1);\n '2016-07-29'")
+  extended = """
+    Arguments:
+      start_date - a date expression.
+      num_days - a numeric expression that represents the number of days to subtract.
```

Comment:
The same one with https://github.com/apache/spark/pull/15513/files/c55ecb60c79694c79a7c60c0c4a759bff6c71fc6#r84830856